src/Components/Components/src/ComponentBase.cs

```diff
@@ -32,7 +32,7 @@ public abstract class ComponentBase : IComponent, IHandleEvent, IHandleAfterRend
 
```

Comment:
The simplifications in this file look great.

@javiercn, are you happy that these changes preserve the behavior of the lifecycle async flow you designed for prerendering?